shelter/dist/rpm/shelter.spec

```diff
@@ -0,0 +1,58 @@
+%define centos_base_release 1
+%define _debugsource_template %{nil}
+
+%global PROJECT inclavare-containers
+%global BIN_DIR /usr/local/bin
+
+Name: shelter
+Version: 0.6.0
+Release: %{centos_base_release}%{?dist}
+Summary: shelter is designed as a remote attestation tool for customer to verify if their workloads are loaded in a specified intel authorized sgx enclaved.
+
+Group: Development/Tools
+License: Apache License 2.0
+URL: https://github.com/alibaba/%{PROJECT}
+Source0: https://github.com/alibaba/%{PROJECT}/archive/v%{version}.tar.gz
+
```

Comment:
You should find a clean ubuntu container and build shelter again and add the BuildRequirements please.

Reply:
Add dependence lib into BuildRequires